eel/eel-editable-label.c

```diff
@@ -1339,9 +1345,18 @@ get_layout_location (EelEditableLabel  *label,
 #endif
     GtkAllocation allocation;
 
+#if !GTK_CHECK_VERSION (3, 0, 0)
     misc = GTK_MISC (label);
+#endif
     widget = GTK_WIDGET (label);
+#if GTK_CHECK_VERSION (3, 0, 0)
+    margin_start = gtk_widget_get_margin_start (widget);
+    margin_end = gtk_widget_get_margin_end (widget);
+    margin_top = gtk_widget_get_margin_top (widget);
+    margin_bottom = gtk_widget_get_margin_bottom (widget);
+#else
     gtk_misc_get_alignment (misc, &xalign, &yalign);
+#endif
 
     if (gtk_widget_get_direction (widget) != GTK_TEXT_DIR_LTR)
         xalign = 1.0 - xalign;
```

Comment:
Well, xalign is used here and both xalign and yalign are used near the end of the function... but in GTK+3 build, xalign and yalign aren't initialized :confused: